ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTest.scala

```diff
@@ -746,39 +747,43 @@ abstract class QueryStoreAndAuthDependentIntegrationTest
 
     "with unknown contractId should return proper error" in withHttpService { fixture =>
       import fixture.encoder
-      val contractIdString = "0" * 66
+      val contractIdString = "0" * 68
```

Comment:
fyi, the reason for this increase is that Canton requires contract IDs to be suffixed. 
i.e., the field `requireV1ContractIdSuffix` is `true` in the Class  `com.daml.lf.engine.preprocessing`